tests/infrastructure/prometheus.go

```diff
@@ -138,14 +138,14 @@ var _ = DescribeInfra("[rfe_id:3187][crit:medium][vendor:cnv-qe@redhat.com][leve
 		}, 10*time.Second, time.Second).Should(BeTrue())
 
 		promIP := ep.Subsets[0].Addresses[0].IP
-		Expect(promIP).ToNot(Equal(0), "could not get Prometheus IP from endpoint")
+		Expect(promIP).ToNot(Equal(""), "could not get Prometheus IP from endpoint")
```

Comment:
Shouldn't we use `BeEmpty()` instead of `""`?

Reply:
This a good question. I think that for strings it's more readable to use `Equal("")`. I guess it's a matter of style. Same as in a regular golang code, where some developer prefers `if s == ""`, while others will use `if len(s) == 0`. The first is more readable, and the second is with better performance. I think that in this case (unit test), I prefer readability over performance.